packages/ses/src/error/fatal-assert.js

```diff
@@ -18,7 +18,9 @@ if (typeof abandon === 'function') {
   /** @param {Error} reason */
   raise = reason => {
     // Check `console` each time `raise` is called.
+    // eslint-disable-next-line no-restricted-globals
```

Comment:
@erights and I expressly decided that lazy binding `console` and `assert` and dynamic dispatch on their methods were the desired, albeit exceptional, behavior. The reason being that SES changes these on lockdown. The machinery to have lockdown incidentally change some shared module state is far too complex to warrant.